### PR TITLE
fix: interactive docs

### PR DESCRIPTION
**Pull request by samedii**

### CODE REVIEW

File: fastapi_oidc/types.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
 from typing import List
 
 from pydantic import BaseModel
-from pydantic import BaseSettings
 from pydantic import Extra
 
 
-class OIDCConfig(BaseSettings):
+class OIDCConfig(BaseModel):
```
Comment (HarryMWinters) [inline fastapi_oidc/types.py:7]: Thanks for cleaning this up!

File: fastapi_oidc/auth.py [hunk 2]
```diff
@@ -83,7 +83,9 @@ def get_auth(
     #    doesn't support OpenIDConnect yet.
     #
     # Some relevant examples https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/
-    oauth2_scheme = OpenIdConnect(openIdConnectUrl=issuer)
```
Comment (HarryMWinters) [inline fastapi_oidc/auth.py:86]: Nice! We can delete the comments above since their now obsolete.

**Review by HarryMWinters — APPROVED** [review]
Looks good! Two non-blocking points:

1. Can we add a test that we call `OpenIdConnect` with the appropriate URL?
2. Can we delete the comments on lines 79-85 since they're now incorrect?

I can release these in 0.0.9 as soon as we merge.

File: fastapi_oidc/auth.py (hunk 2 above)
Comment (samedii) [reply fastapi_oidc/auth.py:86]: Sure, I deleted them in the other PR

**Comment by samedii** [comment]
> Can we add a test that we call OpenIdConnect with the appropriate URL?

There doesn't seem to be something built in but we have this check already:

```
    @cached(TTLCache(1, cache_ttl))
    def discover_auth_server(*_, base_url: str) -> Dict:
        discovery_url = f"{base_url}/.well-known/openid-configuration"
        r = requests.get(discovery_url)
        # If the auth server is failing we can't verify tokens.
        # Soooo panic I guess?
        r.raise_for_status()
        configuration = r.json()
        return configuration
```

What did you have in mind?

**Comment by HarryMWinters** [comment]
I should have been more clear. I meant can we test that we call `fastapi.security.OpenIdConnect` with the correct arguments for r`openIdConnectUrl`.

**Comment by samedii** [comment]
I see but from diving into their code I can't see that any request is made to the url that is provided. I think we would need to start up FastAPI and do some check on the number of available flows which seems overkill?

**Comment by HarryMWinters** [comment]
How about we merge and release this and I open another ticket for what I'm thinking? It's nothing fancy. Just mocking `fastapi.security.OpenIdConnect` and making sure it's called with an appropriate string. See #26

**Comment by HarryMWinters** [comment]
Merged and released in v0.0.9. This also closes #19.

**Pull request merged**